### ISSUE TITLE
[Help wanted] - Background nodes

### ISSUE THREAD

**Issue by cristianocca** [visjs/vis-network#728]
Hello,

I'm trying to render a series of nodes in "background" of the canvas, in a way that they behave as expected on zoom, scroll, and pan actions, but everything else is "ignored" (selection, edge dragging, etc.  Pretty much as a background image, but I still need to be able to reference each "background" node in the coordinate system.

Something like this:
![image](https://user-images.githubusercontent.com/8573267/82352738-c70f3300-99d4-11ea-86f4-584224366294.png)

So in short, is there any way to make nodes completely non interactive?  The `fixed` prop doesn't really help since the node still behaves to other interactions.

**Comment by cristianocca** [comment]
Update: solved it with two canvas overlapped and passing down a `.moveTo` to the back canvas call after each drawing of the foreground canvas.
